<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KLFLibEntryEditor</class>
 <widget class="QWidget" name="KLFLibEntryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>556</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="lyt_KLFLibEntryEditor">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitEntryEditor">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frmPreviews">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="lyt_frmPreviews">
       <item row="0" column="1">
        <widget class="KLFDisplayLabel" name="lblPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>80</height>
          </size>
         </property>
         <property name="pixmap">
          <pixmap>:/pix/pics/nopreview.png</pixmap>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="KLFLatexEdit" name="txtPreviewLatex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="hspc1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="hspc2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="tabsEntry">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCategoryTags">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>322</width>
         <height>257</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="label">
        <string>Category and Tags</string>
       </attribute>
       <layout class="QGridLayout" name="lyt_tabCategoryTags">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lblCategory">
          <property name="text">
           <string>&amp;Category:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>cbxCategory</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="cbxCategory">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="autoCompletion">
           <bool>true</bool>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="lblTags">
          <property name="text">
           <string>Name / &amp;Tags:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>cbxTags</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QComboBox" name="cbxTags">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="autoCompletion">
           <bool>true</bool>
          </property>
          <property name="duplicatesEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="hspc3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btnApplyChanges">
          <property name="text">
           <string>Apply Changes</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="vspcCatTags">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStyle">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>322</width>
         <height>257</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="label">
        <string>Style</string>
       </attribute>
       <layout class="QGridLayout" name="lyt_tabStyle">
        <property name="margin">
         <number>6</number>
        </property>
        <item row="2" column="0" colspan="9">
         <widget class="QLabel" name="lblStyMathModeCaption">
          <property name="text">
           <string>Math Mode:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="9">
         <widget class="QPushButton" name="btnRestoreStyle">
          <property name="text">
           <string>Restore This Style</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <spacer name="vspcStyle">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0" colspan="9">
         <widget class="QLabel" name="lblStyPreamble">
          <property name="text">
           <string>LaTeX Preamble:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="9">
         <widget class="KLFLatexEdit" name="txtStyPreamble">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="9">
         <widget class="QLabel" name="lblStyMathMode">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="hspc6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblStyColors">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Colors:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QFrame" name="frmStyColors">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="lyt_frmStyColors">
           <property name="horizontalSpacing">
            <number>3</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item row="1" column="4">
            <widget class="QLabel" name="lblStyColSlash">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">/</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLabel" name="lblStyColBg">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Background Color</string>
             </property>
             <property name="text">
              <string notr="true">Y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="lblStyColFg">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Foreground Color</string>
             </property>
             <property name="text">
              <string notr="true">X</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="5" colspan="4">
         <widget class="QLabel" name="lblStyDPI">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>16</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string notr="true">1200</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="lblStyDPILbl">
          <property name="text">
           <string>DPI Res.:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="hspc5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="9">
         <widget class="QFrame" name="frmUserScript">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblUserScriptPrompt">
             <property name="text">
              <string>User Script:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblUserScript">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string notr="true">&lt;user script...&gt;</string>
             </property>
             <property name="margin">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLFLatexEdit</class>
   <extends>QTextEdit</extends>
   <header>klflatexedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLFDisplayLabel</class>
   <extends>QLabel</extends>
   <header location="global">klfdisplaylabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtPreviewLatex</tabstop>
  <tabstop>cbxCategory</tabstop>
  <tabstop>cbxTags</tabstop>
  <tabstop>btnApplyChanges</tabstop>
  <tabstop>txtStyPreamble</tabstop>
  <tabstop>btnRestoreStyle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
